src/__tests__/index.js

```diff
@@ -193,6 +193,13 @@ pluginTester({
       `,
     },
 
+    {
+      babelOptions: {filename: __filename},
```

Comment:
We don't need the `filename` because we wont be processing this code.

Let's add: `snapshot: false` to indicate that this code should not be touched. Then let's change the `code` to:

```
import x from "./fixtures/compute-one.js";
```

This will make the `babel-plugin-tester` assert that the code is unchanged.